ci-operator/step-registry/aws/provision/vpc/shared/aws-provision-vpc-shared-commands.sh

```diff
@@ -757,6 +757,14 @@ if [[ "${ADDITIONAL_SUBNETS_COUNT}" -gt 0 ]]; then
   aws_add_param_to_json "AdditionalSubnetsCount" ${ADDITIONAL_SUBNETS_COUNT} "$vpc_params"
 fi
 
+# aws efs single zone cross account the shared vpc subnet zone should be consistent with the cluster
+if [[ ${ENABLE_SINGLE_ZONE} == "true" ]] && [[ "${ZONES_COUNT}" -eq 1 ]]; then
```

Comment:
As this is a special case for EFS, I'd suggest using a particular name, like `EFS_ENABLE_SINGLE_ZONE`, which would be clear for others.

Reply:
Agree, this will be more clear, updated, thank you!